code/modules/antagonists/traitor/objectives/demoralise_crew.dm

```diff
@@ -0,0 +1,53 @@
+#define MAX_CREW_RATIO 0.33
+#define MIN_CREW_DEMORALISED 8
+#define MAX_CREW_DEMORALISED 16
+
+/datum/traitor_objective_category/demoralise
+	name = "Demoralise Crew"
+	objectives = list(
+		/datum/traitor_objective/demoralise/poster = 1,
+		/datum/traitor_objective/demoralise/graffiti = 2,
+	)
```

Comment:
I think you should decrease the weight of this objective to make it less common, so that every traitor is not placing down graffiti and posters.

Reply:
I have applied this, but as the list of potential roundstart objectives can still be quite shallow depending on which jobs are occupied it might not work as expected.  
As this is one of the few with _no_ specific requirement it is always an option, and so it seems like frequently the number of potential objectives is smaller than the number it tries to generate (it tries to pick six!) so it will be picked regardless of weight.  

That said if people get 5 options and this is one, and it goes away after 30 minutes, choosing any of the others is reasonably likely to invalidate this one just through the time it takes them to do it. 
This will go away if we simply add more objectives over time, or if you feel it is necessary I could put some other bespoke coin-flipping solution in generate_objective.